src/test/java/org/betonquest/betonquest/modules/logger/custom/PlayerLogHandlerTest.java

```diff
@@ -25,11 +26,17 @@
  */
 @ExtendWith(BetonQuestLoggerService.class)
 class PlayerLogHandlerTest {
+    /**
+     * The mocked plugin instance.
+     */
+    private final Plugin plugin;
 
     /**
      * Default constructor.
      */
     public PlayerLogHandlerTest() {
+        plugin = mock(Plugin.class);
```

Comment:
The Plugin mock should be reset() after every test.

```java
@BeforeEach
void prepareMocks() {
    when(plugin.getName()).thenReturn("BetonQuest");
}

@AfterEach
void resetMocks() {
    reset(server);
}
```

An alternative would be to use the `@Mock` annotation:
https://javadoc.io/static/org.mockito/mockito-core/4.3.1/org/mockito/Mockito.html#mock_annotation